src/m365/teams/commands/meeting/meeting-transcript-list.spec.ts

```diff
@@ -0,0 +1,258 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { telemetry } from '../../../../telemetry';
+import { Cli } from '../../../../cli/Cli';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { accessToken } from '../../../../utils/accessToken';
+import { pid } from '../../../../utils/pid';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+import { formatting } from '../../../../utils/formatting';
+
+const command: Command = require('./meeting-transcript-list');
+
+describe(commands.MEETING_TRANSCRIPT_LIST, () => {
+  const userId = '68be84bf-a585-4776-80b3-30aa5207aa21';
+  const userName = 'user';
+  const email = 'user@tenant.com';
+  const meetingId = 'MSo5MWZmMmUxNy04NGRlLTQ1NWEtODgxNS01MmIyMTY4M2Y2NGUqMCoqMTk6bWVldGluZ19ZMlEzTlRRMFpEWXRaamMzWkMwMFlUVmhMVGt4TTJJdFpURmtNMkUwTUdGak1qVmpAdGhyZWFkLnYy';
+  const meetingTranscriptsResponse =
+    [
+      {
+        "id": "MSMjMCMjZDAwYWU3NjUtNmM2Yi00NjQxLTgwMWQtMTkzMmFmMjEzNzdh",
+        "createdDateTime": "2021-09-17T06:09:24.8968037Z"
+      },
+      {
+        "id": "MSMjMCMjMzAxNjNhYTctNWRmZi00MjM3LTg5MGQtNWJhYWZjZTZhNWYw",
+        "createdDateTime": "2021-09-16T18:58:58.6760692Z"
+      },
+      {
+        "id": "MSMjMCMjNzU3ODc2ZDYtOTcwMi00MDhkLWFkNDItOTE2ZDNmZjkwZGY4",
+        "createdDateTime": "2021-09-16T18:56:00.9038309Z"
+      }
+    ];
+
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+  let commandInfo: CommandInfo;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
```

Comment:
Let's not forget to add our new stub here:

https://github.com/pnp/cli-microsoft365/blob/54db3109027392637a1b46e42ade8aa852f9681c/src/m365/app/commands/app-open.spec.ts#L29

We should also clear this stub in the `after` method.

Reply:
sorry I am bit lost 🤔 Don't I already do it here 🤔? I mean I have the getProcessName stub in before and after already 🤔

Reply:
Hmm my bad sorry!